tests/integration/article.test.js

```diff
@@ -0,0 +1,135 @@
+import chaiHttp from "chai-http";
+import chai, { expect, should } from "chai";
+import app from "../../index";
+import { newArticle, newUser } from "../testData";
```

Comment:
we are store data for testing purpose in different places, can we have one? @Musigwa